src/clims/services/extensible.py

```diff
@@ -135,6 +135,46 @@ def _register_model(self, name, org, plugin, property_types=None):
         return extensible_type
 
 
+class Location(object):
```

Comment:
Looks good, but I wonder if we can merge it with other code that exists. But that's for another commit!